src/jira/handlers.ts

```
@@ -378,6 +544,58 @@ export class JiraHandlers {
+      const { issueKey, transitionId, comment } = args;
+
+      // Validate issueKey
+      const issueKeyValidation = validateString(issueKey, 'issueKey', {
+        required: true,
+        pattern: /^[A-Z]+-[0-9]+$/,
+      });
+      if (!issueKeyValidation.isValid)
+        return createValidationError(issueKeyValidation.errors, 'transitionJiraIssue', 'jira');
+
+      // Validate transitionId
+      const transitionIdValidation = validateString(transitionId, 'transitionId', { required: true });
```

ESLint

[error] Replace `·required:·true` with `⏎········required:·true,⏎·····`

(prettier/prettier)